bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -258,6 +259,26 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       if c.wv != WaterVapor.ANY
     } yield new Problem(Severity.Warning, s"GMOS is usually unaffected by atmospheric water vapor", "Observations", s.inObsListView(o.band, _.Fixes.fixConditions(c)))
 
+
+    private def gmosNDisperser(b: BlueprintBase, d: GmosNDisperser) = b match {
+      case gn: GmosNBlueprintSpectrosopyBase => gn.disperser == d
+      case _                                 => false
+    }
+    private def gmosSDisperser(b: BlueprintBase, d: GmosSDisperser) = b match {
+      case gs: GmosSBlueprintSpectrosopyBase => gs.disperser == d
+      case _                                 => false
+    }
+
+    private val gmosR600Check = p.proposalClass match {
+      case _: ClassicalProposalClass => Nil
```

Comment:
I think this maybe wrong, you want the other way around. Do the checks if the proposal is classical and ignore (returning Nil) otherwise